quickwit/quickwit-indexing/src/models/split_attrs.rs

```diff
@@ -60,6 +60,9 @@ pub struct SplitAttrs {
 
     // Number of merge operation the split has been through so far.
     pub num_merge_ops: usize,
+
+    /// Timestamp for tracking when the split was finalized.
```

Comment:
Nitpick: if you find a better wording without using finalize as mentionned on the docs, it's nice.

```
/// Timestamp for tracking when indexing ends for the split.
```